bundles/ch.elexis.core.data/src/ch/elexis/data/Rechnung.java

```diff
@@ -597,9 +601,13 @@ public void setTemporaryState(final int temporaryState, TimeTool expiryDate) {
 	 * @param state as defined by {@link InvoiceState#numericValue()}
 	 */
 	public void setStatus(final InvoiceState state) {
```

Comment:
Es muss dann auch `setState` in `ch.elexis.core.model.Invoice` angepasst werden, damit das auch über das neue Modell eingefügt wird.